infos/freizeit.md

```diff
@@ -0,0 +1,79 @@
+---
+category: studienanfänger
+---
+
+# Freizeit
+
+Wir haben für euch alphabetisch ein paar Links zusammengetragen, die
+euch vielleicht ein wenig bei eurer Freizeitplanung helfen.
+
+## Auf dem Sand
+
+Auf dem Sand existiert eine Grillstelle, die im Einvernehmen mit den zuständigen Behörden von Institutsangehörigen genutzt werden kann.
+
+Des Weiteren gibt es ein Volleyballfeld und einen Fußballplatz. Über die [Mailingliste für Sport](/infos/maillists) können sportliche Aktivitäten auf dem Sand koordiniert werden.
+
+## An der Uni
+
+Auch die Uni legt sich mächtig ins Zeug. Ihre Angebote kann man [hier](https://www.uni-tuebingen.de/international/gastwissenschaftler/freizeit.html) finden.
+
+## Fachsprachenzentrum
+
+Die Uni hat ein [Fachsprachenzentrum](https://www.uni-tuebingen.de/fsz) 
+eingerichtet, wo man als Tübinger Student kostenlos (sonst gegen eine
+geringe Gebühr Sprachkurse) belegen kann.
+
+## Fahrplanauskunft
+
+Zur Fahrplanauskunft der Deutschen Bahn geht es
+[hier](https://www.bahn.de).
+
+Fahrpläne für Baden-Württemberg, auch für die Busse in Tübingen, kann
+man bei der [Elektronischen Fahrplanauskunft
+Baden-Württemberg](https://www.efa-bw.de/) abfragen.
+
+Und wer Fahrpläne für andere Bundesländer und viele weitere Länder sucht, wird bei
+[Öffi App](https://offi.schildbach.de/index_de.html) fündig.
+
+## Mitfahrzentralen
+
+Ihr wollt nach Hamburg, Berlin oder München? Nach Stockholm? Oder
+irgendwo sonst hin? Aber es ist zu teuer allein? Dann schaut doch mal
+hier nach:
+
+``` 
+  * [[https://www.mitfahrzentrale.de]]
+  * [[https://www.mitfahrgelegenheit.de]]
+  * [[https://www.drive2day.com]]
+```
```

Comment:
Does the code block make sense here? (I haven't looked at the result yet but it seems a bit strange)